bin/e2e

```diff
@@ -29,4 +29,10 @@ fi
 printf "Starting the test runner...\n\n"
 
 bundle exec rails assets:precompile;
-CYPRESS_RAILS_CYPRESS_OPTS="--config-file cypress.dev.json" RAILS_ENV=test E2E=true bundle exec rake cypress:open
+
+if [ $1 = "--no-seed" ]; then
```

Comment:
I'm explicitly checking `$1` because we only support one argument at the moment.

Reply:
Hi i have been getting ```bin/e2e: line 33: [: =: unary operator expected``` when running e2e test and not all the test is shown 
![image](https://user-images.githubusercontent.com/48720253/127120785-c56e4804-cf2e-4089-b3f2-8cc8995c2fed.png)